newdle/client/src/components/answer/Calendar.js

```diff
@@ -203,19 +206,21 @@ export default function Calendar() {
   const activeDateIndex = optionsByDay.findIndex(({date: timeSlotDate}) =>
     toMoment(timeSlotDate, HTML5_FMT.DATE).isSame(activeDate, 'day')
   );
-
+  const numberOfVisible = isTabletOrMobile ? 1 : 3;
```

Comment:
that's a strange name :D

What about `numDaysVisible` and `numColumns`?

Reply:
I used that because it's also used as prop in `DayCarousel` with that name